styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
+backend
+css
+http 
+https 
```

Comment:
Move all three above acronyms—css, http, https—into `reject.txt`, per previous comment.

Reply:
Vale should not warn about URLs as long as they follow the style rules. See https://github.com/errata-ai/vale/issues/320

If "http" or "https" are used literally as terms and not in URLs, then they should either be inline literals:

```
`http` redirects to `https`
```

...or uppercased to "HTTP" or "HTTPS".